src/components/GameAcceptModal/GameAcceptModal.tsx

```diff
@@ -75,12 +75,18 @@ export class GameAcceptModal extends Modal<Events, GameAcceptModalProperties, {}
             player_color = _("Random");
         }
 
+        const challenger_details: PlayerObjectType = {
+            pro: challenge.pro,
+            rank: challenge.rank,
+            username: challenge.username,
```

Comment:
Let's add: `id: challenge.user_id` too, having that user id makes lookup quicker and nicer on the system if we need to fetch details (which we do for this case since we need the player icon).

Reply:
👍